packages/starter/react-ui-starter/src/Navigation.tsx

```diff
@@ -1,16 +1,42 @@
-import { useWallet } from '@solana/wallet-adapter-react';
+import { WalletNotConnectedError } from '@solana/wallet-adapter-base';
+import { useConnection, useWallet } from '@solana/wallet-adapter-react';
 import { WalletDisconnectButton, WalletMultiButton } from '@solana/wallet-adapter-react-ui';
-import React, { FC } from 'react';
+import { Keypair, SystemProgram, Transaction } from '@solana/web3.js';
+import React, { FC, useCallback } from 'react';
 
 export const Navigation: FC = () => {
-    const { wallet } = useWallet();
+    const { connection } = useConnection();
+
+    const { publicKey, sendTransaction } = useWallet();
+
+    const onClick = useCallback(async () => {
+        if (!publicKey) throw new WalletNotConnectedError();
+
+        const transaction = new Transaction().add(
+            SystemProgram.transfer({
+                fromPubkey: publicKey,
+                toPubkey: Keypair.generate().publicKey,
+                lamports: 1,
+            })
+        );
+
+        const signature = await sendTransaction(transaction, connection);
+        await connection.confirmTransaction(signature, 'confirmed');
+        alert(`done ${signature}`);
+    }, [publicKey, sendTransaction, connection]);
```

Comment:
Just a quick note -- the `example` project (used by the demo) is helpful for testing out wallets like this since it already comes with these things.